src/components/cylc/cylcObject/Menu.vue

```diff
@@ -198,10 +202,13 @@ export default {
       }
       return this.mutations
     },
+    isFamily () {
+      // TODO: better way of checking if a 'task' is actually a family?
```

Comment:
This isn't really safe but ATM all views request `isHeld` so it's fine for the mo.

As of the Graph View PR you can use `node.type` in all situations, it will be one of:

* `task`
* `family`
* `cycle` (can be considered a family although you may need to add `/root` on the end)